src/widgetastic/utils.py

```diff
@@ -686,3 +686,38 @@ def wrap(*args, **kwargs):
             raise StaleElementReferenceException("Couldn't handle it")
 
     return wrap
+
+
+class BasicFillViewStrategy(object):
+    """Used to fill view's widgets by default. It just calls fill for every passed widget
+
+    """
+    def do_fill(self, fill_list):
+        changes = []
+        for widget, value in fill_list:
+            try:
+                changes.append(widget.fill(value))
+            except NotImplementedError:
+                continue
+        return any(changes)
+
+
+class WaitFillViewStrategy(object):
+    """It is used to fill view's widgets where changes in one widget
+    may cause another widget appear.
+
+    New widgets may appear after some delay.
+    So such strategy gives next widget some time to turn up.
+    """
+    def __init__(self, wait_widget='5s'):
```

Comment:
We can go with minimum wait time... like `2s`. User can increase as per requirement.